sdk/communication/azure-communication-jobrouter/CHANGELOG.md

```diff
@@ -121,32 +118,18 @@
 
 #### RouterJob
 - Added `ETag`
-- Added constructor `RouterJob(string jobId)`
 - Added setters for `ChannelId`, `ChannelReference`, `ClassificationPolicyId`, `DispositionCode`, `MatchingMode`, `Priority`, `QueueId`
 
 #### RouterQueue
 - Added `ETag`
-- Added constructor `RouterQueue(string queueId)`
 - Added setters for `DistributionPolicyId`, `ExceptionPolicyId` and `Name`
 
 #### RouterWorker
 - Added `ETag`
-- Added constructor `RouterWorker(string workerId)`
 
 #### BestWorkerMode
 - Added setters to `ScoringRule` and `ScoringRuleOptions`
 
-#### OAuth2WebhookClientCredential
-- Added constructor `OAuth2WebhookClientCredential(string clientId, string clientSecret)`
-
-## 1.0.0-beta.2 (Unreleased)
```

Comment:
In case it tries to validate the Changelog, let's add this back in and move the (Unreleased) back to this.